doc/deploy/android.md

```diff
@@ -1,33 +1,129 @@
-## Deploying to the Google Play Store
-##### Prerequirements
-- You **must** read the [general deploy info](README.md).
-- You **must** have the official PVTrAck signing key. See [the keys doc](android-keys.md) for more info.
-- You **must** have the Android SDK and Android build tools installed (both bundled with Android Studio). We target API level 25 (Nougat, 7.1).
-- You **must** have access to the [Google Play Developer Console](play.google.com/apps/publish) (see @sherson).
-- You *should* have [Android Studio](https://developer.android.com/studio/index.html) installed.
+# Building an Android App to Deploy in Google Play
 
-##### Steps
+## Requirements
+_These instructions assume you are running a Mac. If you're not, the installation
+instructions will be vastly different. You can ask for help or, since you're probably
+running Linux, just translate them like you're probably used to doing._
 
-1. `npm install`
+### General
+- Did you read the [general deploy info](README.md)?
+- Do you have access to the [Google Play Developer Console](play.google.com/apps/publish)?
+ If not, see Adam.
 
-2. Open `config.xml`. On line 2, increment `android-versionCode` with your appropriately chosen new version number.
+### Android Studio
+We use Android Studio to easily manage the Android SDK tools. If you wish, you can attempt
+to download the tools separately, but these instructions will not help you because I found the
+attempt maddening. Plus, Android Studio will keep track of updates for you, which is nice.
+1. Get [Android Studio](https://developer.android.com/studio/)
+1. Use it to download the SDK platforms and tools: 
+  1. Click menu items Android Studio -> Appearance and Behavior -> System Settings -> Android SDK
+  1. In the main window, click "sdk platforms"
+  1. Click 'edit' to choose an sdk location. Follow the prompts. 
+  The default location is fine--keep note of where that is so you can run the tools 
+  from the command line.  Wait for it to download the components.
+  1. Keep waiting
+  1. Check the boxes next to Android 8.0(Oreo) and any others leading up to the latest--you'll see that that's API 
+  level 26, which we target at the moment. 
```

Comment:
I thought we had agreed that we would start with Android 7.1.1 (Nougat) because the API is level 25. Why do we target 26 now?

Reply:
Yeah, you'll want to double-check that. If we only support 26+, then our app only works on [19% of Android phones](https://developer.android.com/about/dashboards/).

Reply:
When I had `<preference name="android-targetSdkVersion" value="26" />` in my `config.xml`,  Google Play wouldn't let me upload the app, saying something like "you must target API level 26 at least because security."
I can try again to get the actual error message.

Reply:
Or [actually](https://developer.android.com/distribute/best-practices/develop/target-sdk)

Reply:
Also, just because the "target API" is 26, that [doesn't mean that older platforms aren't supported.](https://developer.android.com/training/basics/supporting-devices/platforms)

Reply:
Hmm. Welp, I guess that's right. There's nothing we can do about that, but my comment still stands. Only ~20% of Android users will be able to install the latest version.

Reply:
Yup. My mistake. "target" is different than "minimum".

Reply:
Looks like, from an initial reading, that supporting older platforms is a lot like supporting older versions of Ruby--you can backport fixes and have compatibility to a certain extent as long as you're careful about the methods you use.
And since this isn't an Android app, I'm not sure how to go about discerning how to do that.
Presumably, one could leverage Cordova in some way to handle that?

Reply:
Yes, Cordova handles that entirely. Since we're currently using cordova-android 6.x.x, the supported versions are 16-26 and the plugin takes care of setting the `android:minSdkVersion` (in our case to `16`)

Reply:
Ok, cool.